### DISCUSSION TITLE
Templates for .NET SKIA using .NET 5

### DISCUSSION THREAD

**Discussion by microhobby** [unoplatform/uno#8258] [Q&A] [answered]
There are plans for when the templates will be updated to .net6.0?

https://github.com/unoplatform/uno/blob/4d3e8b52742345dd478aae2883df300845d12b97/src/SolutionTemplate/UnoSolutionTemplate/Skia.Gtk/UnoQuickStart.Skia.Gtk.csproj#L6

**Comment by jeromelaban** [comment, marked as answer]
Hi @microhobby! The[ net6.0 templates](https://github.com/unoplatform/uno/blob/4d3e8b52742345dd478aae2883df300845d12b97/src/SolutionTemplate/UnoSolutionTemplate.net6/Skia.Gtk/UnoQuickStart.Skia.Gtk.csproj#L6) can be used for that (`dotnet new unoapp-net6`). We'll merge those in the future, as we're trying to align with .NET 6 mobile targets.
